### ISSUE TITLE
Preprocessing for PyTorch models

### ISSUE THREAD

**Issue by RulinShao** [Trusted-AI/adversarial-robustness-toolbox#846]
**Could not pass means and stds for the channel-first inputs w.r.t. PyTorchClassifier.**
Ideally, two lists of means and stds could be passed directly to the PyTrochClassifier. However, it standardizes the inputs simply using (x - mean) / std such that the mean and the std need to be float numbers. This function may work for TensorFlow since the channel is the last dimension and python will automatically duplicate the values for each channel. However, a bug raises when the input form is (b, c, w, h).

**Possible solution.**
Use repeated or permute function to write the normalization process for the channel-fisrt situation.

**I will appreciate your feedback if there is something I missed or better solutions.**

**Comment by beat-buesser** [comment]
Hi @RulinShao Thank you very much for using ART and raising this issue! We have recognised the issue too. So far the solution described in issue #268 is required for channels-first models, but we are also already preparing an improved solution in pull request #839. If your time allows, could you please try out #839 and let us know what you think?

**Comment by beat-buesser** [comment]
Closing this issue because the solution of #839 has been release in ART 1.6.0
